### ISSUE TITLE
Improve alternative protocols measures

### ISSUE THREAD

**Issue by bkardell** [HTTPArchive/httparchive.org#758]
Currently as part of the "mobile web" chapter of the almanac we track "[alternative protocols](https://almanac.httparchive.org/en/2022/mobile-web#alternative-protocol-links)".  Note that _both_ ends of this are a little strange as there's nothing particular to mobile web about that, but I'm more interested in the other end. Currently we're measuring this on anchor links only, but there are many places where URLs are used and there are different protocols which may or may not be useful in other contexts.

Currently there are a lot of "developing protocols" with different aims and degrees of support through certain browsers or extensions, protocol handlers, etc.  For example, `ipfs:` may (or may not) be relevant as a source for an image/picture or a video or a stylesheet or script or iframe or a form even?  But that's a very particular example, others might be made relevant through a `<link>`  generally or something.  It seems like it might make sense to just catalog all of the places URLs are used and how/which protocols (and ideally store a few sample URLs)..

So, basically just like we do with `<a href=....>` but a map of them rather than just 1?
